### PR TITLE
TINY-8371: Add image import using the asset/inline webpack loader

### PR DESCRIPTION
**Pull request by TheSpyder**

### CODE REVIEW

File: modules/server/src/main/ts/bedrock/compiler/Webpack.ts [hunk 1]
```diff
@@ -54,9 +54,13 @@ const webpackSharedRules = webpackRemap.concat([
     enforce: 'pre'
   },
   {
-    test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/,
+    test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/i,
     type: 'asset/source'
   },
+  {
+    test: /\.(jpe?g|png|gif|svg)$/i,
```
Comment (lnewson) [inline modules/server/src/main/ts/bedrock/compiler/Webpack.ts:61]:
```suggestion
    test: /\.(jpe?g|png|gif|svg|apng|avif|webp|bmp|tiff)$/i,
```
This just adds in more image formats to save us running into issues later (it should be everything we include for TinyMCE).

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Images (svg, png, gif, jpg) can now be imported by tests, and will be available as data URIs using the `asset/inline` webpack loader
```
Comment (lnewson) [inline CHANGELOG.md:10]:
```suggestion
- Images (svg, png, gif, jpg, etc...) can now be imported by tests, and will be available as data URIs using the `asset/inline` webpack loader
```

**Review by lnewson — APPROVED** [review]

File: modules/server/src/main/ts/bedrock/compiler/Webpack.ts (hunk 1 above)
Comment (lnewson) [reply modules/server/src/main/ts/bedrock/compiler/Webpack.ts:61]: Actually there are a few more weird variants like `jpe`, `jfi`, `jif`, and `jfif` if you wanted to include them, but I'm less concerned about them 🤷

Comment (TheSpyder) [reply modules/server/src/main/ts/bedrock/compiler/Webpack.ts:61]: I'm not sure we need any of them, but sure :)

**Review by ltrouton — APPROVED** [review]

**Review by spocke — APPROVED** [review]

**Comment by spocke** [comment]
@TheSpyder Time to get this merged?

**Comment by TheSpyder** [comment]
oops, indeed

**Pull request merged**